package.json

```diff
@@ -102,7 +102,9 @@
     "cypress": "^13.17.0",
     "dotenv": "^16.4.7",
     "eslint": "^9.17.0",
-    "eslint-config-prettier": "^9.1.0",
+    "eslint-config-airbnb": "^19.0.4",
+    "eslint-config-airbnb-typescript": "^18.0.0",
+    "eslint-config-prettier": "^10.0.1",
```

Comment:
Please consider this change.

As far as can tell this adds packages that we actually want to remove.